examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -253,9 +245,9 @@ class FasterRouteActivity : AppCompatActivity(), OnMapReadyCallback {
         bottomSheetBehavior.peekHeight = 0
         fasterRouteAcceptProgress.max = MAX_PROGRESS.toInt()
         startNavigation.setOnClickListener {
+            navigationMapboxMap.enterNavigationMode()
+            navigationMapboxMap.addProgressChangeListener(mapboxNavigation)
```

Comment:
Should users need to think about it? Couldn't we pass `MapboxNavigation` as an optional constructor argument and manage it if provided?

Reply:
I think you suggested previously using the MapboxNavigation reference that's passed in NavigationMapboxMap.addProgressChangeListener, unless I misunderstood you. That seems like a good thing to do since there are other listeners registered there already.